core/common/pom.xml

```diff
@@ -108,6 +108,11 @@
       <artifactId>gmetric4j</artifactId>
       <version>${gmetric4j.version}</version>
     </dependency>
+    <dependency>
```

Comment:
`alluxio-core-server` already has the `protobuf-java` dependency. Would it make sense to remove the dependency in `alluxio-core-server` since it's added to common?